internal/handlers/jsonb1/where.go

```diff
@@ -31,8 +31,8 @@ import (
 func scalar(v any, p *pg.Placeholder) (sql string, args []any, err error) {
 	var arg any
 	switch v := v.(type) {
-	case int32:
-		sql = "to_jsonb(" + p.Next() + "::int4)"
+	case int32, int64:
+		sql = "to_jsonb(" + p.Next() + "::jsonb)"
```

Comment:
That's two different cases, and I think we can use `fjson.Marshal` for int64 (maybe for both)

Reply:
The cases are different but both cases behave the same. 
I tried to use it through `fjson.Marshal` (for int64 and for int32) but it doesn't return values like MongoDB does.